ODBC/OAuthSampleApp.cpp

```diff
@@ -159,12 +160,14 @@ int main(int argc, char **argv)
     std::string clientId;
     std::string clientSecret;
     std::string tokenUrl;
+    std::string scope;
     static struct option long_options[] = {
         {"access-token", required_argument, 0, 'a'},
         {"refresh-token", required_argument, 0, 'b'},
         {"client-id", required_argument, 0, 'c'},
         {"client-secret", required_argument, 0, 'd'},
         {"token-url", required_argument, 0, 'e'},
+        {"scope", required_argument, 0, 'f'},
```

Comment:
Should "f" be added to the getopt_long option string on line 176 as well?

Reply:
Done